### PR TITLE
font-iosevka: update to 4.5.0

### PR DESCRIPTION
**Pull request by georgianfire**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/font-iosevka/template [hunk 1]
```diff
@@ -12,8 +12,8 @@ distfiles="https://raw.githubusercontent.com/be5invis/Iosevka/v${version}/LICENS
  https://github.com/be5invis/Iosevka/releases/download/v${version}/ttc-iosevka-${version}.zip
  https://github.com/be5invis/Iosevka/releases/download/v${version}/ttc-iosevka-slab-${version}.zip"
 checksum="e61c0988bb231a321f14cce1b119a468f279ea86826c32e943ab16dbf08c1ba9
```
Comment (ericonr) [inline srcpkgs/font-iosevka/template:14]: I think this needs to be updated.

Comment (georgianfire) [reply srcpkgs/font-iosevka/template]: You are right. I've updated the checksum. No idea why `xbps-src pkg` didn't complain on my machine though.

Comment (ericonr) [reply srcpkgs/font-iosevka/template]: I think it's because the file name isn't versioned, then it's fulfilled from your local cache. CI helps catch it :)

Comment (georgianfire) [reply srcpkgs/font-iosevka/template]: That makes a lot of sense. Thanks for the explanation!

**Pull request merged**